agent/src/config/configloader.h

```diff
@@ -17,8 +18,12 @@
  * For further information see LICENSE
  */
 
+
 #pragma once
 
+#include "softwarecontainer-common.h"
```

Comment:
Nothing in this header needs anything from `softwarecontainer-common.h`. If it is needed in the cpp file, include it there instead.